### PR TITLE
Fix overflowError: Python int too large to convert to C long

### PR DESCRIPTION
**Pull request by nlpng**
## Motivation and Context

This PR is trying to hot fix the issue of #169

## Description

Main change is in the compute threshold pass (`pass_compute_threshold`) of optimizer.py

## How has this been tested?

The overflow error can be reproduced if the kernel weight is a extremely small value.
Test is plotted as below:
```python
    def test_pass_compute_thresholds_for_huge_threshold_values(self) -> None:
        """Test pass."""
        data1 = np.float32(np.random.rand(1, 2, 2, 3))
        data2 = np.float32(np.random.uniform(10 ** (-30), 10 ** (-40), size=(1, 2, 2, 3)))
        graph1 = self.create_sample_graph(data1, data2)

        pass_compute_thresholds(graph1)

        self.assertEqual(graph1.get_op('conv2').has_thresholds, True,
                         '[Failed] Found threshold of Conv not calculated')

        print("Test pass #8-1 compute_thresholds of enormously small values passed!")
```
By experimentally feeding super small weights of `data2` (around 10**(-30) - 10**(-40)) to the optimization pass of `pass_comput_threshold`, we will get following errors:
```
======================================================================
ERROR: test_pass_compute_thresholds_for_huge_threshold_values (test_optimizer.TestPassComputeThresholds)
Test pass.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/huang/oss_ws/blueoil/dlk/tests/test_optimizer.py", line 463, in test_pass_compute_thresholds_for_huge_threshold_values
    pass_compute_thresholds(graph1)
  File "/Users/huang/oss_ws/blueoil/dlk/python/dlk/core/optimizer.py", line 370, in pass_compute_thresholds
    else int(math.ceil(th_per_ch))
OverflowError: Python int too large to convert to C long

----------------------------------------------------------------------
```
The fix is to take care the threshold value before performing further operations.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by iizukak** [comment]
I think this PR must be merged in v0.4.0

File: dlk/python/dlk/core/optimizer.py [hunk 1]
```diff
@@ -355,6 +355,10 @@ def pass_compute_thresholds(graph: Graph) -> None:
                     bn_nega_idx = [v for v in range(len(bn_scale)) if bn_scale[v] < 0]
             threshold = (trans_th['data'] * np.float64(n)) / (np.float64(max_v) * scaling_factor)
 
+            # take care of threshold values that are larger than 13-bit signed integer
```
Comment (ruimashita) [inline dlk/python/dlk/core/optimizer.py]: This comment is wrong, 12 bit is correct.

https://github.com/blue-oil/blueoil/blob/7f704a253df7dadec6be05bda620f68009f3932f/dlk/python/dlk/core/optimizer.py#L337-L338

Comment (nlpng) [reply dlk/python/dlk/core/optimizer.py]: Thanks for the comment, I thought a 13 bit signed integer has max value of 2^12 - 1 for the sign-bit being the highest bit.

Comment (ruimashita) [reply dlk/python/dlk/core/optimizer.py]: Thanks, Aha, you are right!!
And I just remember, I wrote same comment in previous PR. sorry ~

**Pull request merged**